index.html

```diff
@@ -189,5 +189,6 @@ <h2>
     </ul>
```

Comment:
- [ ] kindly add a hint for the users that the form elements are required , you can simply add `(required)` in the placeholder field example : 
`<input class="input-field" type="email" name="email" placeholder="Email address (required)" required>`